docs/EVM-compatible.md

```diff
@@ -39,6 +39,12 @@ When you send an Ethereum transaction, the transaction is converted to Godwoken
 | GASLIMIT | `block.gaslimit` | 12,500,000 | current block's gas limit |
 | DIFFICULTY | `block.difficulty` | 2,500,000,000,000,000 | current block's difficulty |
 
+### Restriction of memory usage
+
+Polyjuice runs EVM on the ckb-vm. Although there is no limit on memory usage of EVM(well, EVM does have a limit on stack depth which is 1024), ckb-vm can only use 4MB memory at most for now.
```

Comment:
Polyjuice runs EVM on ckb-vm. While EVM has no limit on memory usage (despite the limit of 1024 on stack depth for EVM), ckb-vm can use a maximum of 4MB of memory for now.